scss/treeview/_layout.scss

```diff
@@ -21,6 +21,16 @@
             position: relative;
         }
 
+        // Wrappers
+        .k-top,
+        .k-mid,
+        .k-bot {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            align-content: center;
```

Comment:
Items are intentionally centered in the same way that we have set `white-space: nowrap`.

The change in layout comes from line 63 which adds right margin to the checkbox wrapper, which in terms is `1.5em` wide, which can be improved so it looks fine.